install.sh

```diff
@@ -25,19 +25,30 @@ yarn_get_tarball() {
   else
     url=https://yarnpkg.com/latest.tar.gz
   fi
-  # Get both the tarball and its GPG signature
+
+  download_to_tmp() {
+    if curl --fail -L -o "$1" "$2"; then
+      printf "$green> Downloaded $2.$reset\n"
+    else
+      printf "$red> Failed to download $2.$reset\n"
+      exit 1;
+    fi
+  }
+
   tarball_tmp=`mktemp -t yarn.tar.gz.XXXXXXXXXX`
-  if curl --fail -L -o "$tarball_tmp#1" "$url{,.asc}"; then
-    yarn_verify_integrity $tarball_tmp
+  asc_tmp=`mktemp -t yarn.asc.XXXXXXXXXX`
 
-    printf "$cyan> Extracting to ~/.yarn...$reset\n"
-    mkdir .yarn
-    tar zxf $tarball_tmp -C .yarn --strip 1 # extract tarball
-    rm $tarball_tmp{,.asc}
-  else
-    printf "$red> Failed to download $url.$reset\n"
-    exit 1;
-  fi
+  # Get both the tarball and its GPG signature
+  download_to_tmp $tarball_tmp $url
+  download_to_tmp $asc_tmp $url.asc
```

Comment:
This will download the two files sequentially, whereas the old code downloaded them in parallel. Downloading sequentially will unnecessarily slow down the installation process by adding another round-trip. Could you please do it in a way that still downloads the files in parallel?